### ISSUE TITLE
[BUG] Cannot install or upgrade to 2.0.4, using pnpm

### ISSUE THREAD

**Issue by dflock** [3dmol/3Dmol.js#723]
**Describe the bug**
Looks like the postinstall script hardcodes using npm & webpack and will fail if you're using pnpm and/or aren't using webpack. This was changed here: https://github.com/3dmol/3Dmol.js/commit/95477135851d3e7a8f57f704e9306d62d4f56fd3

**To Reproduce**
```
pnpm add 3dmol
```

**Expected behavior**
It should install correctly.

**Screenshots**

```
node_modules/.pnpm/3dmol@2.0.4_jsdoc@4.0.2/node_modules/3dmol: Running postinstall script, failed in 6.5s
.../node_modules/3dmol postinstall$ npm run build
│ > 3dmol@2.0.4 build
│ > npm run build:dev  && npm run build:prod && npm run generate:tests && npm run doc
│ > 3dmol@2.0.4 build:dev
│ > webpack --config webpack.dev.js --mode development
│ 'webpack' is not recognized as an internal or external command,
│ operable program or batch file.
└─ Failed in 6.5s at C:\Users\DuncanLock\dev\photonic-web-ui\node_modules\.pnpm\3dmol@2.0.4_jsdoc@4.0.2\node_modules\3dmol
 ELIFECYCLE  Command failed with exit code 1.
Error: Command failed with exit code 1: pnpm i
    at makeError (file:///C:/Users/DuncanLock/dev/photonic-web-ui/node_modules/.pnpm/@antfu+ni@0.21.4/node_modules/@antfu/ni/dist/shared/ni.377238f7.mjs:1960:11)
    at handlePromise (file:///C:/Users/DuncanLock/dev/photonic-web-ui/node_modules/.pnpm/@antfu+ni@0.21.4/node_modules/@antfu/ni/dist/shared/ni.377238f7.mjs:2901:26)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async run (file:///C:/Users/DuncanLock/dev/photonic-web-ui/node_modules/.pnpm/@antfu+ni@0.21.4/node_modules/@antfu/ni/dist/shared/ni.377238f7.mjs:6610:3)
    at async check (file:///C:/Users/DuncanLock/dev/photonic-web-ui/node_modules/.pnpm/taze@0.11.2/node_modules/taze/dist/cli.mjs:2134:7)
    at async Object.handler (file:///C:/Users/DuncanLock/dev/photonic-web-ui/node_modules/.pnpm/taze@0.11.2/node_modules/taze/dist/cli.mjs:2424:18) {
  shortMessage: 'Command failed with exit code 1: pnpm i',
  command: 'pnpm i',
  escapedCommand: 'pnpm i',
  exitCode: 1,
  signal: undefined,
  signalDescription: undefined,
  stdout: undefined,
  stderr: undefined,
  failed: true,
  timedOut: false,
  isCanceled: false,
  killed: false
}
 ELIFECYCLE  Command failed with exit code 1.
```

**Desktop (please complete the following information):**
 - OS: Windows 10
 - pnpm 8.7.6

**Additional context**
I'm using pnpm instead of npm and [vite](https://vitejs.dev/) instead of webpack.

**Please close your issue when you feel it has been adequately resolved**

**Comment by dkoes** [comment]
Even worse, `npm install` doesn't work (unless you are running it inside the git checkout, which is all I ever do).  Please verify this is fixed in 2.0.5 and close the issue.

**Comment by dflock** [comment]
This is indeed fixed, thanks!
